opengever/maintenance/scripts/deactivate_spv.py

```diff
@@ -0,0 +1,110 @@
+from Acquisition import aq_inner
+from Acquisition import aq_parent
+from opengever.base.model import create_session
+from opengever.maintenance.debughelpers import setup_app
+from opengever.maintenance.debughelpers import setup_option_parser
+from opengever.maintenance.debughelpers import setup_plone
+from plone import api
+from Products.CMFCore.utils import getToolByName
+from sqlalchemy import MetaData
+from zope.sqlalchemy.datamanager import mark_changed
+import sys
+import transaction
+
+"""Cleanup SPV objects and tables.
+
+    bin/instance run ./scripts/deactivate_spv.py
+
+optional argument:
+  -n : dry run
+  --delete_tables: Also drop the SPV tables from the DB
+
+"""
+
+
+class CleanupSPVAndEnableWordFeature(object):
+
+    def __init__(self, dry_run):
+        self.session = create_session()
+        self.connection = self.session.connection()
+        portal = api.portal.get()
+        self.catalog = getToolByName(portal, 'portal_catalog')
+        self.dry_run = dry_run
+
+    def __call__(self, delete_tables=False):
+        self.remove_plone_content()
+        if delete_tables:
+            self.clear_sql_tables()
+
+    def remove_plone_content(self):
+        portal_types = (
+            'opengever.meeting.committee',
+            'opengever.meeting.committeecontainer',
+            'opengever.meeting.period',
+            'opengever.meeting.submittedproposal',
+            'opengever.meeting.proposal',
+            'opengever.meeting.proposaltemplate',
+            'opengever.meeting.sablontemplate',
+            'opengever.meeting.meetingdossier',
+            )
+
+        for portal_type in portal_types:
+            print("Starting deletion of {}".format(portal_type))
+            for brain in self.catalog.unrestrictedSearchResults({'portal_type': portal_type}):
+                obj = brain.getObject()
+                print("deleting {}".format(obj.absolute_url()))
+                aq_parent(aq_inner(obj)).manage_delObjects([obj.getId()])
+
+    def clear_sql_tables(self):
+        print("Starting deletion of tables")
+        metadata = MetaData(self.connection, reflect=True)
+        for table_name in (
+                'agendaitems',  # => proposals, meetings
```

Comment:
i think you should add `'generateddocuments'` and `'meetings'`

Reply:
Yes, sorry, I misread an upgrade step which I thought deleted the table... But no, it's still here